appveyor.yml

```diff
@@ -26,44 +33,96 @@ services:
+        $env:PYTEST_TARGETS=$env:TESTABLE_PYTEST_TARGETS.Replace(" ", ",")
+        $rake_targets=$env:TESTABLE_RAKE_TARGETS.Split(' ')
+
+        echo "This is a master build, running tests on all targets."
+      }
+  - cmd: if "%TEST_RAKE%"=="true" bundle install
+  - cmd: if "%TEST_RAKE%"=="true" bundle package
+  - cmd: git clone -b %DD_AGENT_BRANCH% https://github.com/DataDog/dd-agent.git c:\projects\dd-agent
+  - cmd: if not exist %PIP_CACHE% mkdir %PIP_CACHE%
   - cmd: appveyor-retry powershell If (-Not (Test-Path $env:PYWIN_PATH)) {(new-object net.webclient).DownloadFile("$env:PYWIN32_URL", "$env:PYWIN_PATH")}
   - "%PYTHON%/Scripts/easy_install.exe %PYWIN_PATH%"
-  - ps: mkdir -p $(python -m site --user-site)
-  - ps: echo "C:\projects\dd-agent" | out-file "$(python -m site --user-site)/datadog-agent.pth" -encoding ASCII
-  - ps: (& "$env:PYTHON/python.exe" -m pip install --upgrade pip)
-  - ps: '& "$env:PYTHON/Scripts/pip.exe" install -r c:\projects\dd-agent\requirements.txt'
-  # Remove the adodbapi module shipped with pywin32: it conflicts with the pip-installed adodbapi
-  - ps: rm $env:PYTHON/lib/site-packages/$env:PYWIN32_INSTALL_DIR/adodbapi/__init__.py
-  - ps: rm $env:PYTHON/lib/site-packages/$env:PYWIN32_INSTALL_DIR/adodbapi/__init__.pyc
-  - ps: (& "$env:PYTHON/Scripts/pip.exe" install -U virtualenv)
-  - ps: (& "$env:PYTHON/Scripts/pip.exe" install tox)
-  - ps: choco install docker-compose
-  - ps: (& "$env:PYTHON/Scripts/pip.exe" install invoke)
-  - ps: (& "$env:PYTHON/Scripts/pip.exe" install pip-tools)
-  - ps: (& "$env:PYTHON/Scripts/pip.exe" install packaging)
-  - cd datadog_checks_base
-  - ps: (& "$env:PYTHON/Scripts/pip.exe" install .)
-  - cd ..
-  - cd sqlserver
-  - ps: (& "$env:PYTHON/Scripts/pip.exe" install -r requirements.txt)
-  - ps: (& "$env:PYTHON/Scripts/pip.exe" install .)
-  - cd ..
-  - cd windows_service
-  - ps: (& "$env:PYTHON/Scripts/pip.exe" install .)
-  - cd ..
-  - cd wmi_check
-  - ps: (& "$env:PYTHON/Scripts/pip.exe" install .)
-  - cd ..
+  - ps: >-
+        mkdir -p $(python -m site --user-site)
+        echo "C:\projects\dd-agent" | out-file "$(python -m site --user-site)/datadog-agent.pth" -encoding ASCII
+  - ps: >-
+
+        # Upgrade pip and install Agent requirements
+        (& "$env:PYTHON/python.exe" -m pip install --upgrade pip) 
+        (& "$env:PYTHON/Scripts/pip.exe" install -r c:\projects\dd-agent\requirements.txt)
+
+        # Remove the adodbapi module shipped with pywin32: it conflicts with the pip-installed adodbapi
+        rm $env:PYTHON/lib/site-packages/$env:PYWIN32_INSTALL_DIR/adodbapi/__init__.py
+        rm $env:PYTHON/lib/site-packages/$env:PYWIN32_INSTALL_DIR/adodbapi/__init__.pyc
+
+        # Install test dependencies
+        (& "$env:PYTHON/Scripts/pip.exe" install -U virtualenv setuptools)
+        choco install docker-compose
+        (& "$env:PYTHON/Scripts/pip.exe" install invoke pip-tools packaging tox twine cookiecutter colorama requests)
```

Comment:
This installs all the modules in `requirements-dev.txt` except for `docker-compose`, which is installed by Chocolatey for compatibility reasons.